src/main/java/com/google/api/codegen/discovery/config/SampleConfig.java

```diff
@@ -80,6 +80,10 @@
   @JsonProperty("apiVersion")
   public abstract String apiVersion();
 
+  /** Returns true if clients should be qualified with the API version. */
```

Comment:
I'm not quite sure what you mean, but I've changed it to be less specific than the `true` case.